src/app/components/demo-components.service.ts

```diff
@@ -382,7 +382,7 @@ export class SkyDemoComponentsService {
         name: 'Phone field',
         icon: 'phone',
         // tslint:disable-next-line
-        summary: `The phone field module allows users to validate international phone numbers.`,
+        summary: `The phone field module creates a field and dropdown to validate international phone numbers.`,
```

Comment:
This will not be true with the UI updates that are being made. We can chat around what we want the verbiage here to be.

Reply:
I'm changing "creates a field and dropdown" to "creates a button and text input" both here and in the intro on the the component topic. (I'm also making additional edits to the intro and the rest of the topic.)